.github/workflows/tests.yml

```
@@ -16,7 +16,7 @@ name: tests
 on: [pull_request]
 
 jobs:
-  build:
+  lint:
     name: lint
     runs-on: ubuntu-latest
     steps:
```

```suggestion
    if: "${{ github.event.action != 'labeled' || github.event.label.name == 'tests: run' }}"
    name: Run lint
    runs-on: ubuntu-latest
    steps:
      - name: Remove PR Label
        if: "${{ github.event.action == 'labeled' && github.event.label.name == 'tests: run' }}"
        uses: actions/github-script@v6
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
          script: |
            try {
              await github.rest.issues.removeLabel({
                name: 'tests: run',
                owner: context.repo.owner,
                repo: context.repo.repo,
                issue_number: context.payload.pull_request.number
              });
            } catch (e) {
              console.log('Failed to remove label. Another job may have already removed it!');
            }
```
this will allow label to trigger the lint job as well, might also consider moving the lint job to a `lint.yaml` file of its own, since it won't need to be triggered on the periodic or continuous builds.

---

probably will want to add this to `coverage.yml` too so that coverage check is run when label trigger happens.